Source/src/modules/ModuleWindow.cpp

```diff
@@ -41,6 +43,20 @@ bool Hachiko::ModuleWindow::Init()
             flags |= SDL_WINDOW_RESIZABLE;
         }
 
+        SDL_GL_SetAttribute(SDL_GL_ACCELERATED_VISUAL, 1);
```

Comment:
These were on create context of ModuleRender, but while trying to set the depth component to 32bits I discovered that some of these SetAttributes should be called before creating the window, proven by some other answers about sdl problems, so I moved them into the place before window creation. If you want, we can move them into a method of ModuleWindow.

Reply:
Maybe they can be on a method of moduleRender that is called from here? Idk as long as they are grouped with a bit of cotnext on what is their purpose it will be fine

Reply:
Done 👍